.github/workflows/ci.yml

```diff
@@ -46,10 +46,12 @@ jobs:
         run: |
           tox
         env:
+          DJANGO_CONFIGURATION: DevelopmentConfiguration
```

Comment:
Why is this needed?

Reply:
Without it, we get an environment error: https://github.com/OpenImaging/miqa/runs/6011730594?check_suite_focus=true